synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -118,18 +142,12 @@ export class SynopsysToolsParameter {
       };
     }
 
-    const buildReason =
-      taskLib.getVariable(AZURE_ENVIRONMENT_VARIABLES.AZURE_BUILD_REASON) || "";
-
-    if (
-      parseToBoolean(inputs.POLARIS_REPORTS_SARIF_CREATE) ||
-      parseToBoolean(inputs.POLARIS_REPORTS_SARIF_CREATE_CLASSIC_EDITOR)
-    ) {
-      if (buildReason !== AZURE_BUILD_REASON.PULL_REQUEST) {
+    if (parseToBoolean(inputs.POLARIS_REPORTS_SARIF_CREATE)) {
+      if (!isPullRequest()) {
```

Comment:
can we store the isPullRequest() value in some variable and reuse here?

Reply:
updated